<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::KeyboardProfileDialog</class>
 <widget class="QDialog" name="Otter::KeyboardProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="actionsTab">
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <layout class="QHBoxLayout" name="hactionsLayout">
       <item>
        <layout class="QVBoxLayout" name="actionsViewLayout">
         <item>
          <widget class="Otter::LineEditWidget" name="filterLineEditWidget">
           <property name="placeholderText">
            <string>Searchâ€¦</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Otter::ItemViewWidget" name="actionsViewWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>200</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="actionsButtonsLayout" stretch="0,0,1">
         <item>
          <widget class="QPushButton" name="addActionButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeActionButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="actionsSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="informationTab">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <layout class="QFormLayout" name="informationLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>titleLineEditWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="buddy">
          <cstring>descriptionLineEditWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>Version:</string>
         </property>
         <property name="buddy">
          <cstring>versionLineEditWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="authorLabel">
         <property name="text">
          <string>Author:</string>
         </property>
         <property name="buddy">
          <cstring>authorLineEditWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Otter::LineEditWidget" name="titleLineEditWidget"/>
       </item>
       <item row="1" column="1">
        <widget class="Otter::LineEditWidget" name="descriptionLineEditWidget"/>
       </item>
       <item row="2" column="1">
        <widget class="Otter::LineEditWidget" name="versionLineEditWidget"/>
       </item>
       <item row="3" column="1">
        <widget class="Otter::LineEditWidget" name="authorLineEditWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::KeyboardProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::KeyboardProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
